formatTest/typeCheckedTests/input/jsx.re

```diff
@@ -59,6 +59,15 @@ module Namespace = {
     };
 };
 
+module Optional = {
+    let createElement ::optional children => {
```

Comment:
I'm surprised you don't get a warning

> Warning 43: the label optional is not optional.

Since it's not marked as optional `::optional=?` at definition time.

You might want to add a couple of test cases of these forms

```
let createElement ::notActuallyOptional children =>
let createElement ::optional=? children =>
let createElement ::optional="DefaultValue" children =>
```